File: 3019-furthest-point-from-origin/3019-furthest-point-from-origin.ts
function furthestDistanceFromOrigin(moves: string): number {
    let L = 0
    let R = 0
    let _ = 0
    for(let i=0;i<moves.length;i++){
        if(moves[i]=="L"){
            L++
        } else if (moves[i]=="R"){
            R++
        } else {
            _++
        }
    }
    if(L>R){
        return L-R+_
    } else {
        return R-L+_
    }
};

File: 3019-furthest-point-from-origin/3019-furthest-point-from-origin.py
class Solution:
    def furthestDistanceFromOrigin(self, moves: str) -> int:
        L = 0
        R = 0
        _ = 0

        for ch in moves:
            if ch=="L":
                L+=1
            elif ch=="R":
                R+=1
            else:
                _+=1

        if L>R:
            return L-R+_
        else:
            return R-L+_